package.json

```diff
@@ -105,5 +122,7 @@
     "typescript": "^4.9.5",
     "vscode-test": "^1.3.0"
   },
-  "dependencies": {}
-}
+  "dependencies": {
+    "yarn": "^1.22.22"
```

Comment:
This repo assumes `yarn` is installed globally, same as `npm` in most (current) node projects. This explicit dependency should be removed.